app/models/experimentation/notifications_experiment.rb

```diff
@@ -2,6 +2,7 @@ module Experimentation
   class NotificationsExperiment < Experiment
     include ActiveSupport::Benchmarkable
     BATCH_SIZE = 1000
+    EXCLUDED_BLOCKS = YAML.load_file("config/data/blocks_excluded_from_experiments.yml")
```

Comment:
We have country specific env var files now, see https://github.com/simpledotorg/simple-server/blob/master/.env.IN.  

You could add the ENV var there, and then load it from there to keep our configuration more consistent.